dom.bs

```diff
@@ -5523,6 +5548,22 @@ or <var>A</var>'s <a for=tree>root</a> is a <a for=/>shadow root</a> whose
 <a for=ShadowRoot>mode</a> is "<code>open</code>" and <a for=DocumentFragment>host</a> is an
 <a>unclosed node</a> of <var>B</var>.
 
+<p>To <dfn export lt="retarget|retargeting">retarget</dfn> a <var>node</var> against a
+<var>base</var>, run these steps:</p>
+
+<ol>
+ <li>Let <var>target</var> be <var>node</var>.
```

Comment:
We don't really need this step, right? And instead of "run these steps" above we could say "repeat these steps until they return a node"?